test/SparqlExpressionTestHelpers.h

```diff
@@ -90,12 +94,40 @@ struct TestContext {
         localVocab.getIndexAndAddIfNotContained(iri("<notInVocabD>")));
     notInVocabAelpha = Id::makeFromLocalVocabIndex(
         localVocab.getIndexAndAddIfNotContained(lit("notInVocabÄlpha")));
+    notInVocabAelpha = Id::makeFromLocalVocabIndex(
+        localVocab.getIndexAndAddIfNotContained(lit("notInVocabÄlpha")));
     notInVocabIri =
         Id::makeFromLocalVocabIndex(localVocab.getIndexAndAddIfNotContained(
             iri("<http://www.w3.org/1999/02/22-rdf-syntax-ns#langString>")));
     notInVocabIriLit =
         Id::makeFromLocalVocabIndex(localVocab.getIndexAndAddIfNotContained(
             lit("http://www.w3.org/1999/02/22-rdf-syntax-ns#langString")));
+    berlin = Id::makeFromLocalVocabIndex(
+        localVocab.getIndexAndAddIfNotContained(lit("berlin")));
```

Comment:
please factor this repetitive code out into a local lambda `addLocalLiteral`. 
And while you are at it, also add `addLocalIri` and then make this whole initialization easier to read and maintain.